Tests/Runtime/MeshOps/DeleteElementsTests.cs

```diff
@@ -6,18 +6,29 @@
 using UnityEngine.ProBuilder;
 using UnityEngine.ProBuilder.MeshOperations;
 using UnityEngine.ProBuilder.Tests.Framework;
+using System.Collections.Generic;
 
 static class DeleteElementsTests
 {
     static System.Random s_Random = new System.Random();
 
-    static ShapeType[] shapeTypes
-    {
-        get { return (ShapeType[])typeof(ShapeType).GetEnumValues(); }
+    public static List<Type> shapeTypes {
```

Comment:
this code is repeated 3 times below, could you put it in a function and use typecache ?

Reply:
I can't use TypeCache since it would need UnityEditor and we don't want that in those cases, but I'll put it in a global function